addons/project/models/project_milestone.py

```diff
@@ -12,12 +12,13 @@ class ProjectMilestone(models.Model):
     _name = 'project.milestone'
     _description = "Project Milestone"
     _inherit = ['mail.thread']
-    _order = 'deadline, is_reached desc, name'
+    _order = 'sequence'
 
     def _get_default_project_id(self):
         return self.env.context.get('default_project_id') or self.env.context.get('active_id')
 
     name = fields.Char(required=True)
+    sequence = fields.Integer('Sequence', default=0)
```

Comment:
`default=0` is already the default value of a integer field.

Instead, we could give `10` as default value as we did in task for instance.